### ISSUE TITLE
Minor CSS improvements for mobiles and filters

### ISSUE THREAD

**Issue by krzysztofspilka** [handsontable/handsontable#7659]
### Description

I would like to improve the user experience on mobile devices by fixing some issues with editors. Also, I noticed a small misalignment in filters in the context menu, which also should be addressed.

1) Mobile: We should show the borders of the editor while being in the `edit` mode. Here's how it looks now (iPhone 7, Chrome):

![IMG_5483](https://user-images.githubusercontent.com/8048526/112614414-87e61580-8e21-11eb-8036-bdc9c851664b.PNG)

2) Mobile/Chrome issue: We should make the `double-tap to edit` more precise. I noticed that especially on handsontable.com/examples it requires to tap 3 times (very fast) in order to enter the `edit` mode. I turned off all the plugins (from the checkbox list) but it didn't help. Tested on iPhone 7, Chrome. On Safari it works fine.

3) Desktop/Filters/Context menu: On https://handsontable.com/docs/8.3.2/demo-filtering.html I noticed that the "Filter by condition" submenu is being displayed 1px too low which cause this tiny visual glitch that resembles double border at the top of it (see the screenshot below for reference). I recommend positioning the `htMenu` 1px higher by changing its `top` position.
<img width="260" alt="Screen Shot 2021-03-26 at 10 58 12" src="https://user-images.githubusercontent.com/8048526/112615258-8ec15800-8e22-11eb-84da-c04144ca43f9.png">

### Your environment
* Handsontable version: 8.3.2
* Browser Name and version: Chrome 89
* Operating System: macOS Mojave

**Comment by aninde** [comment]
Related to https://github.com/handsontable/handsontable/issues/6961.

**Comment by Osky772** [comment]
Thank you @aninde, I've checked points 1 and 3 from the issue description and it's related only to the iOS. These problems don't exist on Android devices.

**Comment by Osky772** [comment]
Unfortunately, I tried to fix this iOS bug with a double-tap, but I can't do this. This is what I've discovered:

The iOS has a special delay for double-tap which is `350` ms (https://webkit.org/blog/5610/more-responsive-tapping-on-ios/) and the second tap during this delay isn't possible. Moreover, the HOT had z different delays for clearing double-tap timeout, which I aligned to the same value in https://github.com/handsontable/handsontable/commit/7fbdec0b03538d7bdb41880e29001bf92cb0ac79. Before that change, on iOS it was possible to edit a cell only between `350` and `500` ms. Now it's possible to edit after `350` and before `1000` ms. The perfect solution though would be a fast double-tap without any delay. I've achieved it by adding `event.preventDefault()` in https://github.com/handsontable/handsontable/blob/fa685dc8c194c0269b893d202580ce7091ed8322/src/3rdparty/walkontable/src/event.js#L293-L300 but it has a drawback that disables scrolling.

I tried to preserve `touchmove` event and disable `touchstart` somehow, but I don't know how to do this. @wojciechczerniak  @nikersify maybe you have some thought about this?
